addons/binding/org.openhab.binding.onkyo/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:onkyo:config">
+		<parameter name="ipAddress" type="text" required="true">
+			<label>Network Address</label>
+			<description>The IP or host name of the Onkyo Receiver</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="port" type="integer" min="1" max="65335">
```

Comment:
Advanced done. My idea was that port isn't required, but there is a default value. And this is how it works now. If user doesn't give port or port is empty in Paper UI, default value is provided to binding.  Eclipse SmartHome specification is little bit loose here, so don't know how this is thought to work?